apps/client/src/common/hooks/useSocket.ts

```diff
@@ -3,6 +3,12 @@ import { RuntimeStore, SimpleDirection, SimplePlayback } from 'ontime-types';
 import { useRuntimeStore } from '../stores/runtime';
 import { socketSendJson } from '../utils/socket';
 
+export const setClientRemote = {
+  setIdentify: (payload: { target: string; state: boolean }) => socketSendJson('set-client-identify', payload),
+  setRedirect: (payload: { target: string; path: string }) => socketSendJson('set-client-redirect', payload),
+  setRename: (payload: { target: string; name: string }) => socketSendJson('set-client-rename', payload),
+};
+
```

Comment:
suggestion: would it be better if this was next to the `setClientName` function?